src/applications/vaos/services/appointment/transformers.js

```diff
@@ -467,6 +495,45 @@ export function transformConfirmedAppointments(appointments) {
   });
 }
 
+function transformPendingCCAppointment(appt) {
```

Comment:
This function appears almost identical to `transformPendingAppointment`. What do you think about modifying that one instead of creating this new one?

Reply:
Done.